### DISCUSSION TITLE
清除缓存后重新打开，显示The property Modified is not supported for OrderBy.

### DISCUSSION THREAD

**Discussion by congyoubing777** [AlistGo/alist#332] [Q&A]
群晖用docker搭建完成，绑定onedrive后，发现不显示文件夹，后台清除缓存后，回到首页发现错误：The property Modified is not supported for OrderBy。

**Comment by xhofe** [comment]
OneDrive for Business不支持某些排序，建议选择name排序，具体可查看https://docs.microsoft.com/zh-cn/onedrive/developer/rest-api/concepts/optional-query-parameters?view=odsp-graph-online

**Reply by congyoubing777** [reply]
原来如此，谢谢大佬！！

**Reply by ZXGU12** [reply]
建议写到文档里（

**Reply by PANDAZZ1108** [reply]
对，建议写在文档里。
